app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    components = command.split()
+    command_, source_file, destination_file = command.split()
```

Comment:
You can use components instead of command.split() here
```suggestion
    command_, source_file, destination_file = components
```